### PR TITLE
Docs: How to restore a standalone PacketFence server ?

### PR DESCRIPTION
**Pull request by nqb**
# Description
Add doc to describe how to restore a standalone PacketFence server.

# Impacts
- PacketFence Installation Guide

# Delete branch after merge
YES

### CODE REVIEW

File: docs/PacketFence_Installation_Guide.asciidoc [hunk 1]
```diff
@@ -5176,6 +5177,54 @@ NOTE: On Debian the service will be called `mysql` and on CentOS 7 `mariadb`. Ma
 
 Should the service fail to start, make sure you look into the MySQL error logs.
 
+
+[appendix]
+How to restore a standalone PacketFence server ?
```
Comment (extrafu) [inline docs/PacketFence_Installation_Guide.asciidoc:5180]: Remove space before ?

File: docs/PacketFence_Installation_Guide.asciidoc [hunk 2]
```diff
@@ -5176,6 +5177,54 @@ NOTE: On Debian the service will be called `mysql` and on CentOS 7 `mariadb`. Ma
...
+------------------------------------------------
+
+Following steps assume:
```
Comment (extrafu) [inline docs/PacketFence_Installation_Guide.asciidoc]: For the restore procedure, we make the following assumptions:

**Comment by nqb** [comment]
You can review

File: docs/PacketFence_Installation_Guide.asciidoc [hunk 3]
```diff
@@ -5176,6 +5176,57 @@ NOTE: On Debian the service will be called `mysql` and on CentOS 7 `mariadb`. Ma
...
+- `/usr/local/pf` removed
+
+Procedure:
+----
+# cd /root/backup/
+# BACKUP_FILES=packetfence-files-dump-2018-10-01_00h30.tgz
+# BACKUP_DB_FILE=packetfence-db-dump-2018-10-01_00h30.sql.gz
+----
+
+Reinstall pf (to avoid creating var directories by hand):
```
Comment (extrafu) [inline docs/PacketFence_Installation_Guide.asciidoc]: pf -> PacketFence

**Pull request merged**